front/pages/api/w/[wId]/data_sources/index.ts

```diff
@@ -109,7 +109,7 @@ async function handler(
 
       const dataSourceProviderId = "openai";
       const dataSourceModelId = "text-embedding-ada-002";
-      const dataSourceMaxChunkSize = 256;
+      const dataSourceMaxChunkSize = 512;
```

Comment:
I think we can move to 512 for all new data sources no?

Reply:
I had gathered from the slack discussion we were moving to a dual chunking with no mix and a switch to go from 1 to the other, to avoid comparing 512 chunks with 256 chunks scores
But if you think it's fine then ok

Reply:
This setting is just for new data sources and won't impact existing data sources.

Yes you may have mix-match of 256 and 512 across data sources but that I think is fiiiiiiiine